src/util/common.ts

```diff
@@ -17,7 +17,9 @@ import { IDocumentData } from "../types/IDocument";
 
 // A function to abbreviate a given count number into a string format with an appropriate abbreviation suffix
 export function abbreviateCount(count: number): string {
-    if (count < 1000) {
+    if(count < 0){
+        return "?";
```

Comment:
Wondering why do we need it?

Reply:
Because we can't have document count when we do KV filtering

Reply:
Cool!